alpha/lib/model/entry.php

```diff
@@ -3342,4 +3359,62 @@ public function getUserNames() {
 		
 		return implode(" ", $userNames);
 	}
+
+	public function getcapabilities()
+	{
+
+		$capabilitiesUnserialized = $this->getFromCustomData(self::CAPABILITIES);
+		if (is_null($capabilitiesUnserialized))
+		{
+			return "";
+		}
+		else
+		{
+			$capabilitiesArr = unserialize($capabilitiesUnserialized);
```

Comment:
I don't understand why you serialize and unserialize, save it as is, the entire custom-data object is serialized anyway.